controllers/csi/gc/logs.go

```diff
@@ -0,0 +1,100 @@
+package csigc
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/spf13/afero"
+)
+
+const (
+	maxLogFolderSizeBytes = 300000
+	maxNumberOfLogFiles   = 1000
+	maxLogAge             = 14 * 24 * time.Hour
+)
+
+type logFileInfo struct {
+	UnusedVolumeIDs []os.FileInfo
+	NumberOfFiles   int64
+	OverallSize     int64
+}
+
+func (gc *CSIGarbageCollector) runLogGarbageCollection(tenantUUID string) {
+	fs := &afero.Afero{Fs: gc.fs}
```

Comment:
This can be avoided as all methods have access to `gc.fs` anyways. Where needed, Afero utility functions can be used.

Reply:
Fixed